src/java/org/apache/cassandra/service/StorageProxy.java

```diff
@@ -1738,13 +1745,16 @@ private static float calculateResultRowsUsingEstimatedKeys(ColumnFamilyStore cfs
     public static List<Row> getRangeSlice(AbstractRangeCommand command, ConsistencyLevel consistency_level)
     throws UnavailableException, ReadFailureException, ReadTimeoutException
     {
+        consistency_level = maybeCoerceReadConsistencyLevel(consistency_level);
```

Comment:
This feature is for now heavily relying on a manual audit of the code that confirmed that we hit all the read path we wanted. I am wondering if there is a way we could add some tests to assert this remains true in case of any modifications to those code path in the future